listenbrainz/webserver/static/js/src/BrainzPlayer.tsx

```diff
@@ -280,44 +295,6 @@ export default class BrainzPlayer extends React.Component<
     });
   };
 
-  getSourceIndexByListenData = (listen: Listen | JSPFTrack): number => {
```

Comment:
This was moved to the datasources themselves that now implement an `isListenFromThisService` method